docs/Examples/Form.example.purs

```diff
@@ -199,7 +201,9 @@ type User =
   , descriptiveCheckbox :: Boolean
   , height :: Validated String
   , addresses :: Validated (Array Address)
-  , pets :: Validated (Array Pet)
+  , pets :: Validated (Array (ValidatedNewtype Pet))
+  -- ^ validation helpers like `setModified` need a little assistance getting
+  --     into Newtypes like `Pet`, hence this `ValidatedNewtype` wrapper
```

Comment:
What's this `Validated` wrapper around the pets array do?

Reply:
This is because we're validating also the whole array with
```purescript
F.warn (\pets -> Monoid.guard (Array.null pets) (pure "You should adopt a pet."))
```

Reply:
ohh right, ty!